backend/server/src/args.rs

```diff
@@ -0,0 +1,13 @@
+//! This module defines the command line arguments Tobira accepts.
+
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+/// Video portal for Opencast.
```

Comment:
What is this comment describing? :thinking:

Reply:
It's used by `structopt` to generate a description for the app, shown with `--help`.

Reply:
Oof, okay? That's a weird choice. Dunno, looks very strange to me. Should the struct be called `App`, then, maybe? :thinking:

Like, the `Args` are obviously not the video portal ...

Reply:
Would you prefer this?

```
#[structopt(about = "Video portal for Opencast")]
```

Reply:
I think so, yes. :thinking: